### PR TITLE
Update README with details on mysql setup

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

**Comment by Barthelemy** [comment]
please wait,  I will add some doc for the qct

File: README.md [hunk 1]
```diff
@@ -6,20 +6,26 @@ A CC7 machine or a mac.
 
 ## Setup
 
-We use alibuild, see complete instructions [here](https://alice-doc.github.io/alice-analysis-tutorial/building/) and install O2 (no need to `init` it, just `build` it).
+1. Install O2
+  * We use alibuild, see complete instructions [here](https://alice-doc.github.io/alice-analysis-tutorial/building/) and install O2 (no need to `init` it, just `build` it).
+  
+2. Install the MySQL/MariaDB development package 
+  * CC7 : sudo yum install mariadb-devel
```
Comment (knopers8) [inline README.md]: Actually I needed to install `mariadb-server` in order to get this working. Installing `devel` doesn't help (checked).

**Comment by Barthelemy** [comment]
ok, thanks for your comment, fixed

File: README.md [hunk 2]
```diff
@@ -132,6 +138,16 @@ qcSpy
 qcCheckerLauncher -c file:///absolute/path/to/example-default.ini -n checker_0
 ```
 
+### QCG (Web QC GUI)
+
+To run :
+```
+alienv enter qcg/latest-o2-dataflow
```
Comment (knopers8) [inline README.md]: A minor inconsequence, that we could fix: in line 18 you use `o2` defaults in the build command for qcg and here you enter environment with `latest-o2-dataflow` defaults.

Comment (Barthelemy) [reply README.md]: valid point, fixed

**Comment by Barthelemy** [comment]
the error has nothing to do

**Pull request merged**